### PR TITLE
Update Gitter links to Slack

### LINKED ISSUE #700: Remove links to Gitter
From https://github.com/open-telemetry/community#special-interest-groups

### PR DESCRIPTION
**Pull request by dhruv-vora**
Updated the links of the discussion channels from Gitter to Slack.

Fixes #700

### CODE REVIEW

**Comment by dhruv-vora** [comment]
I could not find an appropriate Slack discussion channel for the OTel Errors Working Group meeting. This is the [Gitter channel](https://gitter.im/open-telemetry/error-events-wg). Could someone point me to it and I will update it

**Comment by tedsuo** [comment]
Thanks for doing this! @dhruv-vora the Errors group is completed, we can remove it.

**Comment by dhruv-vora** [comment]
@tedsuo Should I remove the entire row of the Errors group from the table, or just the Gitter link?

**Comment by tedsuo** [comment]
Yes, please remove the group.

**Comment by dhruv-vora** [comment]
@tedsuo could you please review these changes

**Review by alolita — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -106,15 +106,14 @@ GoLang: SDK|Every Thursday alternating between 10:00 and 15:00 PDT|[Google Doc](
...
-Rust: SDK|Every other week on Tuesday alternating between 07:00 and 15:00 PT|[Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)|[Zoom](https://zoom.us/j/8203130519)|[Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
-Specification: General|Every Tuesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/1-bCYkN-DWJq4jw1ybaDZYYmx-WAe6HnwfWbkm8d57v8/edit?usp=sharing)|[Zoom](https://zoom.us/j/6729396170)|[Gitter](https://gitter.im/open-telemetry/opentelemetry-specification)
-Specification: Errors|Every Thursday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/1HgUI69rBridFzCXxXuTjQrkG6jb-YcFQnZjPcyBcK1U/edit#)|[Zoom](https://zoom.us/j/6729396170)|[Gitter](https://gitter.im/open-telemetry/error-events-wg)|
+Rust: SDK|Every other week on Tuesday alternating between 07:00 and 15:00 PT|[Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)|[Zoom](https://zoom.us/j/8203130519)|[Slack](https://cloud-native.slack.com/archives/CTS0TCGVA)|
```
Comment (SergeyKanzhelev) [inline README.md:112]: this link is wrong

Comment (dhruv-vora) [reply README.md:112]: @SergeyKanzhelev could you point me to the right link

Comment (SergeyKanzhelev) [reply README.md:112]: CC: @open-telemetry/rust-approvers @open-telemetry/rust-maintainers 

I don't see the channel created. Perhaps we can merge PR without this change or just create a channel?

**Review by SergeyKanzhelev — CHANGES_REQUESTED** [review]
blocking pr for now

File: README.md (hunk 1 above)
Comment (TommyCpp) [reply README.md:112]: I think the current consensus within Rust SIG is to stick with Gitter. 

If we need to create a slack channel for Rust SIG. I can keep an eye on it.

**Review by SergeyKanzhelev — APPROVED** [review]

**Comment by SergeyKanzhelev** [comment]
thank you!

**Pull request merged**